repos/fdbt-site/src/components/TapPricingRow.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { ReactElement } from 'react';
+import { ErrorInfo, MultiTap } from '../interfaces';
+import FormElementWrapper, { FormGroupWrapper } from './FormElementWrapper';
+
+interface TapPricingRowProps {
+    numberOfTapsToDisplay: number;
+    errors: ErrorInfo[];
+    userInput: MultiTap;
+}
+
+export const renderTable = (index: number, errors: ErrorInfo[], userInput: MultiTap = {}): ReactElement => (
+    <fieldset key={index} className="govuk-fieldset">
+        <legend className="govuk-fieldset__legend govuk-visually-hidden">{`Enter details for tap ${index + 1}`}</legend>
+        <div className="flex-container">
+            <div className="govuk-!-margin-left-4 govuk-!-margin-right-2">
+                <div className="govuk-form-group ">
+                    <label className="govuk-label" htmlFor={`multi-tap-number-${index}`}>
+                        <span className="govuk-label" id={`multi-tap-number-hint-${index}`}>
+                            Tap number {index + 1}
+                        </span>{' '}
+                    </label>
+                </div>
+            </div>
+            <div className="govuk-!-margin-left-2 govuk-!-margin-right-2">
+                <FormGroupWrapper errors={errors} errorIds={[`multi-tap-price-${index}`]}>
+                    <>
+                        <label className="govuk-label" htmlFor={`multi-tap-price-${index}`}>
+                            <span className="govuk-visually-hidden">{`Tap Price - Price ${index + 1}`}</span>
+                            <span aria-hidden>Price</span>
+                        </label>
+                        <span className="govuk-hint" id={`tap-price-hint-${index}`}>
+                            {index === 0 ? 'e.g. 2.99' : ''}
```

Comment:
Could we do this instead?

{index === 0 ? <span className="govuk-hint" id={`tap-price-hint-${index}`}>
                           e.g. 2.99
</span> : null}